File: RolePlayGame/java/fixtures/my/examples/CalculatorTwoFixture.java
package my.examples;

import fit.Fixture;

public class CalculatorTwoFixture extends Fixture {
	private double numerator;
	private double denominator;
	private double result;

	public void numerator(double aNumerator){
		this.numerator = aNumerator;
	}
	
	public void denominator(double aDenominator){
		this.denominator = aDenominator;
	}
	
	public void quotient(){
		Calculator calculator = new Calculator();
		this.result = calculator.quotientFor(numerator, denominator);
	}
	
	public double result(){
		return this.result;
	}
}

File: RolePlayGame/dotnet/fixtures/my.examples/CalculatorTwoFixture.cs
using fit;

namespace my.examples {

	public class CalculatorTwoFixture : Fixture {
		private double numerator;
		private double denominator;
		private double result;
	
		public double Numerator { set { numerator = value; } }
		
		public double Denominator { set { denominator = value; } }
		
		public void Quotient(){
			Calculator calculator = new Calculator(); // system under test
			result = calculator.QuotientFor(numerator, denominator);
		}
		
		public double Result { get { return result; } }
	}
}